### ISSUE TITLE
would like to be able to get nfs connection count per node if possible

### ISSUE THREAD

**Issue by TyfoidKId** [NetApp/harvest#1283]
We already have CIFS connection counts. I'm going to be asked about NFS connection counts at some point. I'm trying to build some Grafana screens for our monitoring reporting people.  FWIW everything so far is outstanding with Harvest 2.0 and then the new nabox3 layered on top.

Thank you for your time

**Comment by Falcon667** [comment]
Good point. Or would it be possible active nfs connections per volume/lif like „nfs connected clients show“ ?
